lib/src/geom/modifier/modifier.dart

```diff
@@ -9,6 +15,27 @@ import 'package:graphic/src/dataflow/tuple.dart';
 abstract class Modifier {
   @override
   bool operator ==(Object other) => other is Modifier;
+
+  GeomModifierOp<GeomModifier> toGeomModifierOp(ToGeomModifierOpParams params);
+}
+
+/// The only use of this class to pass parameters to [Modifier.toGeomModifierOp].
+/// A class is used, instead of named arguments, to avoid breaking changes when
+/// adding new fields to these parameters.
+class ToGeomModifierOpParams {
```

Comment:
Decided to use a class here so we can add new parameters in the future without creating a breaking changes for consumers of this API. 

Unfortunately Dart's named arguments require consumers to specify all parameters, if ones that aren't used, so adding a new argument to named arguments would be a breaking change.